artifacts/definitions/SUSE/Linux/Events/SystemLogins.yml

```diff
@@ -0,0 +1,144 @@
+name: SUSE.Linux.Events.SystemLogins
+description: |
+   Pulls all the system login and logout events from auditd logs.
+   
+type: CLIENT_EVENT
+
+parameters:
+   - name: AuditLogFile
+     default: /var/log/audit/audit.log
```

Comment:
I thought there was an audispd backend that vrr had that meant we didn't need to tail this effectively?

Reply:
Can you please point me to the code/documentation on how audispd working in conjunction with vrr? I can't seem to find anything in the linux-security-sensor repo that is related to audispd.

Reply:
https://github.com/SUSE/linux-security-sensor/blob/main/artifacts/definitions/Linux/Events/ProcessExecutions.yaml